dolphinscheduler-ui/src/views/security/alarm-instance-manage/use-detail.ts

```diff
@@ -41,6 +46,27 @@ export function useDetail(getFormValues: Function) {
     return JSON.stringify(json)
   }
 
+  const testSend = async (json?: IJsonItem[]) => {
+    const values = getFormValues()
+
+    if (status.testing) return
+    status.testing = true
+    try {
+      const res = await testAlertPluginInstance({
+        pluginDefineId: values.pluginDefineId,
+        pluginInstanceParams: formatParams(json, values)
+      })
+      window.$message.success(
+        res
+          ? res.msg
+          : `${t('security.alarm_instance.test_send')} ${t('home.success')}`
+      )
+      status.testing = false
+    } catch (err) {
```

Comment:
please delete err, because it is not used.

Reply:
Done